R/pkg/R/stats.R

```diff
@@ -134,9 +129,7 @@ setMethod("freqItems", signature(x = "SparkDataFrame", cols = "character"),
             collect(dataFrame(sct))
           })
 
-#' approxQuantile
-#'
-#' Calculates the approximate quantiles of a numerical column of a SparkDataFrame.
+#' approxQuantile - Calculates the approximate quantiles of a numerical column of a SparkDataFrame.
```

Comment:
Fixed, sorry I missed one `approxQuantile`. thanks for testing it out.
I thought completely arbitrary do-not-have-newline would be hard for someone new to follow, so I added `@description` instead to make it clear in case of copy/paste and someone added a new empty line.